### DISCUSSION TITLE
Unexpected search behavior

### DISCUSSION THREAD

**Discussion by scoch5** [craftcms/cms#14978] [General]
When I do a search (craft.entries().search(searchQuery).orderBy('score')) the matrix blocks also appear among the search results. I would expect that by default the search would not also include these sections within Craft cms.

**Comment by brandonkelly** [comment]
This is expected; you can pass the `section` param to limit the results to only entries of certain sections.

That said, it ought to be easily possible to limit the results to only top-level section entries, without listing out each section. So I’ve just added support for passing `'*'` to the `section` param, for Craft 5.2 (0b7895507d717c4c2835b01edc203db122dd10e1).

```twig
{% set query = craft.entries()
  .section('*')
  .search(searchQuery)
  .orderBy('score') %}
```

**Reply by scoch5** [reply]
Thank you @brandonkelly!

**Reply by brandonkelly** [reply]
Craft 5.2.0-beta.1 is out with that change.
